specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/workspace.json

```
@@ -1353,7 +1353,12 @@
       "properties": {
         "initialWorkspaceAdminObjectId": {
           "description": "AAD object ID of initial workspace admin",
-          "type": "string"
+          "type": "string",
+          "x-ms-mutability": [
+            "create",
+            "read",
+            "update"
+          ]
```

This change only adds verbosity -- it is equivalent to omitting "x-ms-mutability".
```suggestion
          "type": "string"
```